packages/cli/src/utils/startUp.js

```diff
@@ -30,8 +30,8 @@ async function startUp({ context, options = {}, command }) {
   context.command = command.name();
   context.commandLineOptions = options;
   context.print = createPrint();
-  context.baseDirectory = path.resolve(options.baseDirectory || process.cwd());
 
+  context.baseDirectory = path.resolve(options.baseDirectory || process.cwd());
```

Comment:
Does `baseDirectory` stay the same? Or does it also now fall under the `directories` object?

Reply:
We need it here so we know where to read your `lowdefy.yaml` file from